Foundation/URLSession/URLSessionConfiguration.swift

```diff
@@ -148,11 +148,9 @@ open class URLSessionConfiguration : NSObject, NSCopying {
     }
 
     open class var ephemeral: URLSessionConfiguration {
-        // Return a new ephemeral URLSessionConfiguration every time this property is invoked
-        // TODO: urlCredentialStorage should also be ephemeral/in-memory
-        // URLCredentialStorage is still unimplemented
         let ephemeralConfiguration = URLSessionConfiguration.default.copy() as! URLSessionConfiguration
         ephemeralConfiguration.httpCookieStorage = .ephemeralStorage()
+        ephemeralConfiguration.urlCredentialStorage = URLCredentialStorage() // All credential storage in s-c-f are ephemeral.
```

Comment:
Maybe change to `NOTE: all credential storage in s-c-f is ephemeral at the moment`, since someone might implement `.permanent` at some point for each platform.

Reply:
Maybe I should just have a marker `init(ephemeral:)` initializer, so it's clear to a future maintainer that some storages must be ephemeral.